File: hardToTest/MaxIncrease/MaxIncrease.py
def max_increase(sales):
    n = len(sales)
    max_increase = 0

    for x in range(n - 1):
        for y in range(x + 1, n):
            increase = sales[y] - sales[x]
            if increase > max_increase:
                max_increase = increase

    return max_increase

File: hardToTest/MaxIncrease/MaxIncrease.java
public class MaxIncrease {
    public static int maxIncrease(int[] sales) {
        int n = sales.length;
        int maxIncrease = 0;  // 初始最大增幅为0

        // 遍历所有可能的 x 和 y，计算 A[y] - A[x]
        for (int x = 0; x < n - 1; x++) {
            for (int y = x + 1; y < n; y++) {
                // 如果 A[y] - A[x] 更大，更新最大增幅
                int increase = sales[y] - sales[x];
                if (increase > maxIncrease) {
                    maxIncrease = increase;
                }
            }
        }
        return maxIncrease;  // 返回最大增幅
    }
